reference/5-api-publisher/API-Connection-Management.md

```diff
@@ -0,0 +1,64 @@
+
+```json
+{
+  "configurationStore": {
+    "provider": "postgreSql",
+    "sqlServer": {
+      "connectionString": "Server=(local);Database=EdFi_API_Publisher_Configuration;Trusted_Connection=True"
+    },
+    "postgreSql": {
+      "connectionString": "Server=localhost;Port=5432;Database=edfi_api_publisher_configuration",
+      "encryptionPassword": ""
+    },
+    "awsParameterStore": {
+      "Profile": "default",
+      "Region": "us-east-1"
+    },
+    "plainText": { }
+  }
+}
+```
+
+Currently there are four configuration store implementations available:
+
+| Setting Value                    | Implementation                                                                           | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| -------------------------------- | ---------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <nobr>`sqlServer`</nobr>         | [Configuration Store for SQL Server](ConfigurationStore/Sql-Server.md)                   | Reads API connection details from and writes successfully published changed version numbers to a SQL Server configuration table. Enables management of individual connection settings with encryption for securely storing keys and secrets using the built-in support for symmetric key encryption in SQL Server.                                                                                                                                                                                                                                                                                        |
```

Comment:
In general we want to stay away from using HTML. There is little likelihood of these words being broken across multiple lines, so let's remove the `<nobr>` HTML tag please.

Reply:
The lines break after removing <nobr> that's why I left it

Reply:
The lines look fine in this table after removing `<nobr>`.